kernel/mod_typing.ml

```diff
@@ -287,7 +287,7 @@ let rec translate_mse_funct env ~is_mod mp inl mse = function
       if is_mod then sign,reso
       else subst_modtype_signature_and_resolver (mp_from_mexpr mse) mp sign reso in
     sign, NoFunctor alg, reso, cst
-  |(mbid, ty) :: params ->
+  |(mbid, ty, inl) :: params ->
```

Comment:
Does this really need to be the same `inl` as the one from the toplevel argument? Naively I think that below, `translate_modtype` should take the inlining of the parameter and `translate_mse_funct` the toplevel one... (I can't say I understand this code though.

Reply:
You're right, there is a typo. This should be called differently to not override the `inl` recursively passed to `translate_mse_funct`. Thanks for the careful reading. Fixed.